data/defikingdoms_dex/tokens.py

```diff
@@ -0,0 +1,21 @@
+import json
+
+f = open('liquidity_pools.json')
+y = json.loads(f.read())
+
+dups = []
+
+print("token_address,token_name,token_symbol,decimals")
+for lpPool in y["data"]["pairs"]:
+    if (lpPool["token0"]["id"] in dups) == False:
+        print(lpPool["token0"]["id"]+","+lpPool["token0"]["name"]+","+lpPool["token0"]["symbol"]+","+lpPool["token0"]["decimals"])
+        dups.append(lpPool["token0"]["id"])
+        
+    if (lpPool["token1"]["id"] in dups) == False:
+        print(lpPool["token1"]["id"]+","+lpPool["token1"]["name"]+","+lpPool["token1"]["symbol"]+","+lpPool["token1"]["decimals"])
+        dups.append(lpPool["token1"]["id"])
+
```

Comment:
what do you think about using sets here instead of lists? if can just dedup it

Reply:
for some reason when i tried to use sets, they get re-ordered and I can't get them to behave..